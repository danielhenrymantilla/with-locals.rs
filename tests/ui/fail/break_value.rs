include!("../prelude.rs");

#[with]
fn break_value ()
{
    #[with] fn f () -> &'ref () { &() }

    for n in 0 .. {
        let _: &'ref _ = f();
        if n >= 5 {
            break ();
        }
    }
}
