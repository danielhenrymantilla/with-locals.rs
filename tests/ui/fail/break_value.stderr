error[E0571]: `break` with value from a `for` loop
  --> $DIR/break_value.rs:11:13
   |
11 |             break ();
   |             ^^^^^^^^ can only break with a value inside `loop` or breakable block
   |
help: instead, use `break` on its own without a value inside this `for` loop
   |
11 |             break;
   |             ^^^^^
